app/controllers/api/v1x0/mixins/rbac_mixin.rb

```diff
@@ -0,0 +1,40 @@
+module Api
+  module V1x0
+    module Mixins
+      module RBACMixin
+        def create_access_check
```

Comment:
Please add comments why some use `permission_check` while others use `resource_check`. This is a mixin therefore it should be more generic.
Create, destroy, and update likely need the same permission, while read may require a different one.